tests/__init__.py

```diff
@@ -40,3 +44,20 @@ def returns_distinct_classes(self):
         Each call returns a completely new class.
         """
         assert simple_class() is not simple_class()
+
+
+def create_class(attrs):
```

Comment:
The naming here is a bit confusing given `simple_class` above.  Maybe `hyp_create_class`?  Or make it private